.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,41 @@
+name: Lighthouse CI
+
+on:
+  pull_request:
+    branches: [ master ]
+    types: [opened, synchronize, reopened]
+
+jobs:
+  lighthouse:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Wait for the Netlify Preview
+        # https://github.com/JakePartusch/wait-for-netlify-action
+        uses: jakepartusch/wait-for-netlify-action@v1
+        id: netlify
+        with:
+          site_name: 'layer5'
+
+      - name: Audit URLs using Lighthouse for desktop
+        # https://github.com/treosh/lighthouse-ci-action
+        uses: treosh/lighthouse-ci-action@v9
+        with:
+          urls: ${{ steps.netlify.outputs.url }}
```

Comment:
@Nikhil-Ladha is there any way using which we can share outputs from other workflows? I am a bit stuck on that here 😶 

CC @Chadha93

Reply:
You can use the `find-comment` action, only if you have the PR number for which the workflow is running.
Or, if that's not possible then another way is to save the deploy URL as an artifact in the `preview-site` workflow and then download that artifact here and get the URL from it.

If the first one is feasible, then that is preferred.